opendatalakehouse/03_predict.md

```diff
@@ -1,4 +1,4 @@
-# 04_predict
+# 03_predict
```

Comment:
even in this file we have to remove visualise here as well!
https://github.com/cloudera/cdp-trial-patterns/pull/5/files#diff-3252e488f941c66a32318b3f4e3eb88193ec44ee5268ad0eb2a2f15ef5812ca5R16